src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,141 @@
+package duke.ui;
+
+import duke.task.Task;
+import duke.task.TaskList;
+
+/**
+ * Encapsulates the text user interface of Duke, which deals
+ * with providing the messages sent from Duke to the user.
+ */
+public class Ui {
```

Comment:
I really like the names of the methods in this class, they are all descriptive and intuitive!